sdk/attestation/azure-security-attestation/inc/azure/attestation/attestation_client.hpp

```diff
@@ -0,0 +1,114 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#pragma once
+
+#include <azure/core/internal/json/json.hpp>
+#include <shared_mutex>
+#include <string>
+
+#include "attestation_client_options.hpp"
+#include <azure/attestation/attestation_client_models.hpp>
+#include <azure/core/internal/http/pipeline.hpp>
+
+namespace Azure { namespace Security { namespace Attestation {
+
+  using namespace Azure::Core;
```

Comment:
We don't use `using namespace ...` from headers, as it gets applied into customer code.

We only allow `using namespace ..` from cpp files